### PR TITLE
Add dockerfile and bind server to 0.0.0.0

### PR DESCRIPTION
**Pull request by derekkraan**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM alpine:3.6
+
+RUN apk --update add openjdk7-jre tar
```
Comment (lvonk) [inline Dockerfile]: We are using jdk 8 to compile https://github.com/zilverline/apache-fop-server/blob/master/pom.xml#L235 so can we bump this to java 8?

Comment (derekkraan) [reply Dockerfile]: Ok. Updated and new image pushed (`zilverline/fop:1.7`)

File: README.md [hunk 2]
```diff
@@ -11,10 +11,16 @@ Testing is done using the [Dispatch](http://dispatch.databinder.net/) HTTP clien
     mvn assembly:assembly
 
 ### How to use
-Download one of the tar.gz from the downloads page
-    
-    tar xvzf apache-fop-server-[version].tar.gz
-    java -cp apache-fop-server.jar:lib/* org.zilverline.fop.FopServer
+Use the docker image:
+```sh
+docker run zilverline/fop:1.7
```
Comment (lvonk) [inline README.md]: 👍

**Comment by lvonk** [comment]
:shipit:

**Pull request merged**